packages/workspace/src/tasks-runner/default-tasks-runner.ts

```diff
@@ -98,9 +54,8 @@ async function runAllTasks(
     initiatingProject?: string;
     projectGraph: ProjectGraph;
     nxJson: NxJsonConfiguration;
-    hideCachedOutput?: boolean;
   }
-): Promise<Array<{ task: Task; type: any; success: boolean }>> {
+): Promise<Promise<{ [id: string]: TaskStatus }>> {
```

Comment:
I couldn't see any immediate benefit in having the nested `Promise` here, is there one?

If it were changed to be:

```ts
return await orchestrator.run();
```

and

```ts
return runAllTasks(tasks, options, context);
```

it would keep the signature simpler and be functionally equivalent in this case I think?